### PR TITLE
Time Delay Statistics

### PR DESCRIPTION
**Pull request by hamza-sadruddin**
Please have a look at the files regarding the Task_E ("Time difference between the sender and receiver package handling"). For this, additional function output_time_statistics has been added in simulator_renaissance.cpp, which calculates the time delay statistics between the sender and receiver correspondence and also calculates the max, min, and avg time delay.  (Hamza & Saad). Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
... subnets just pass the packets after a time delay. However, in order to simulate the unreliability of the network, only 95% of the packets will be passed in each of the subnet, i.e. 5% of the data will be lost through the subnet. There is a good documentation for the installation installation of cadmium installation on different platforms like windows and Linux and about Alternate Bit Protocol Simulator. Following the link for more guide.[Installation Guide and Documentation](https://github.com/hamza-sadruddin/AlternateBitProtocolSimulator/tree/master/doc). We are focuing on the on the proper documentation. reviewing the code, following software documentation and conventions, code standards along with the optimization of the new feature to imporve the simulator.The new feature which implemented by generating the output files in Comma separated values format which is the most widely used fromat in the field of Data Science and analytics and also human readible format. The enhancement also include the time delay statistics between sender and receiver correspondance along with the maximum, minimum and average delay by following the proper comments and coding conventions and standards.
```
Comment (jrajeev459) [inline README.md]: Lines < 80 characters:  you should follow coding conventions.

File: src/simulator_renaissance.cpp [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+                     * starting from this parameter position till the end-of-line.
+                     */
+                    string component_check = payload.substr(payload.find("model") + 6);
+                    counter = 0;				      /**<  counter is reset */
+                    /**
+                     * The following do-while loop works until the first 0 is encountered in arr
+                     * starting after the 0th element position*/
+                    do {
+                        val_pos = payload.find("{", arr[counter]); /**< Find the first "{" position starting from arr value position */
+                        string value = payload.substr(val_pos + 1, payload.find("}", /**< Returns the substring between {} */
```
Comment (jrajeev459) [inline src/simulator_renaissance.cpp]: Lines < 80 characters:  you should follow coding conventions. please do it for other lines as well.

File: README.md (hunk 1 above)
Comment (khanyousefzai) [reply README.md]: Done. You can check our file in edit mode

File: src/simulator_renaissance.cpp (hunk 2 above)
Comment (khanyousefzai) [reply src/simulator_renaissance.cpp]: done

**Review by jrajeev459 — APPROVED** [review]
Everything is good. Thank you

**Review by abhinavachanta — APPROVED** [review]
All the files look good.thanks

**Pull request merged**